README.md

```diff
@@ -15,51 +15,92 @@ When you are done:
 #### Loops & Lists
 - [Max Value](https://repl.it/@Admin7/maxvalue)
 ```python
-# copy & paste your solution here
+def return_max(l):
+  if len(l) < 1:
+    return 'None'
+  else:
+    return(max(l))
 ```
-
 - [Min Value](https://repl.it/@Admin7/minvalue)
+
 ```python
-# copy & paste your solution here
+
+def return_max(l):
+  if len(l) < 1:
+    return 'None'
+  else:
+    return(min(l))
 ```
 
 - [Extend](https://repl.it/@Admin7/extendlist)
 ```python
-# copy & paste your solution here
+def return_extended(l, iter):
+  for i in iter:
+    l.append(i)
```

Comment:
Awesome! If we didn't want to loop ourselves, we could also use the `extend` list method to append an iterable at the end of a list.